src/parser/druidsql.pegjs

```diff
@@ -173,6 +185,21 @@ FromClause = fromKeyword:FromToken postFrom:_ tableHead:(Alias/SqlRef) tableTail
   }
 }
 
+JoinClause = joinType:JoinType postJoinType:_ joinKeyword:JoinToken postJoinKeyword:_? table:(Alias/SqlRef) postJoinTable:_  onKeyword:OnToken postOn:_? onExpression:Expression
+{
+  return {
+    joinType: joinType,
+    postJoinType: postJoinType,
+    joinKeyword: joinKeyword,
+    postJoinKeyword: postJoinKeyword,
+    table: table,
+    postJoinTable: postJoinTable,
```

Comment:
Could you please suffix everything that is a space with `Spacing`

Reply:
JoinClause is not a class so all all the spacing is eventually added to the inner spacing of the SqlQueryObject, I can add it for just the rule though.